### PR TITLE
NewEventContent integrated with paywall

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This is an intermediate PR that puts in more of the scaffolding for the paywall ticket purchases. `NewEventContent` is not live yet, so there are no user-visible changes.

Summary of changes:
- `NewEventContent` in TS (not totally, just enough to make some things easier. There's still a lot of `any` in there)
- The paywall app is now the piece responsible for ticket purchases. The config will need to be improved, but right now it is possible to buy a ticket and the page will transition to the ticket info.
- Various pieces of the event page have been split into distinct components. For the most part they are unchanged, this just keeps it easy to only care about one piece of the page at a time.
- Currently no QR code is shown; that's a big enough change that I think it should be a separate PR.

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: tickets/src/components/content/NewEventContent.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+  paywallScriptUrl = (): string => {
+    const {
+      config: { env },
+    } = this.props
+    const extension = '/static/unlock.1.0.min.js'
+
+    // TODO: add test url?
+    const baseUrls: { [key: string]: string } = {
+      dev: 'http://localhost:3001',
```
Comment (akeem) [inline tickets/src/components/content/NewEventContent.tsx]: Should these be in here? I am not  in the loop on how the application is written but could we pass these in from elsewhere?

File: tickets/src/components/content/NewEventContent.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          <Column>
+            {paywallStatus === PaywallStatus.Locked && (
+              <PurchaseTicket
+                onClick={() => {
+                  if (typeof window !== 'undefined') {
+                    window.unlockProtocol &&
+                      window.unlockProtocol.loadCheckoutModal()
+                  }
+                }}
+                keyPrice="0.01"
```
Comment (akeem) [inline tickets/src/components/content/NewEventContent.tsx]: Is this place holder code until the next pr?

File: tickets/src/components/content/NewEventContent.tsx (hunk 1 above)
Comment (cnasc) [reply tickets/src/components/content/NewEventContent.tsx]: Good question, I'm not 100% sure how we handle this elsewhere, but I'll look into it

File: tickets/src/components/content/NewEventContent.tsx (hunk 2 above)
Comment (cnasc) [reply tickets/src/components/content/NewEventContent.tsx]: It is, I plan on either
- getting the lock and key info out of the paywall directly
- or just hooking up to the lock information tickets currently passes through redux

**Comment by julien51** [comment]
Hum, this PR broke the UI of the ticket creation :/
image

Probbably not worth a revert but we need to fix this ASAP.

File: tickets/src/components/content/NewEventContent.tsx (hunk 1 above)
Comment (julien51) [reply tickets/src/components/content/NewEventContent.tsx]: huh, this clearly does not belong there :(

**Comment by cnasc** [comment]
Not sure I see the breakage, what is it more specifically?

File: tickets/src/components/content/NewEventContent.tsx (hunk 2 above)
Comment (julien51) [reply tickets/src/components/content/NewEventContent.tsx]: Yikes. This should have not been approved/merged as is.

**Pull request merged**